python/ray/air/_internal/device_manager/device_manager.py

```diff
@@ -0,0 +1,19 @@
+from abc import ABC, abstractmethod
+
+
+class DeviceManager(ABC):
```

Comment:
Let's also make `get_devices`, `is_device_available`, `cleanup` abstract methods of the base class.